added context (#6736)
open-cluster-management/rhacm-docs: business_continuity/backup_restore/backup_validate.adoc

 
When you set the `cluster-backup` option to `true` on the `MultiClusterHub` resource, {mce-short} installs the cluster backup and restore operator Helm chart that is named the `cluster-backup-chart`. This chart then installs the `backup-restore-enabled` and `backup-restore-auto-import` policies. Use these policies to view information about issues with your backup and restore components.

> *Note:* A hub cluster is automatically imported and managed by itself if it uses the `local-cluster` managed cluster. If you disable this by setting `disableHubSelfManagement=true` on the `MultiClusterHub` resource, the `backup-restore-enabled` policy is not placed on the hub cluster, and the policy templates do not produce any reports.

Author: @birsanv Brandi said the original sentence here was confusing. Am I right that the hub cluster only is automatically imported and manages itself _if_ the hub cluster uses a local-cluster managed cluster?

Reviewer: @jc-berger we describe a `local-cluster` as a hub that is self-managed. The installation creates a namespace called local-cluster that is reserved for a self-managed hub. (It could be that was not understood at the time of building the content or that this means something else.) 

I see this described different ways, but the simplest is to look at the setting and what it means, then describe it that way:

A hub cluster is automatically imported and self-managed as the `local-cluster`. If you disable self-management by setting `disableHubSelfManagement` to `true`...

Now it could be that we are trying to say: if it uses the `local-cluster` _namespace_?

> *Note:* A hub cluster is automatically imported and managed by itself if it uses the `local-cluster` managed cluster. If you disable this by setting `disableHubSelfManagement=true` on the `MultiClusterHub` resource, the `backup-restore-enabled` policy is not placed on the hub cluster, and the policy templates do not produce any reports.

Reviewer 2: Based on Brandi's suggestions
```suggestion
*Note:* A hub cluster is automatically imported and self-managed by as the `local-cluster`. If you disable self-management by setting `disableHubSelfManagement` to `true` on the `MultiClusterHub` resource, the `backup-restore-enabled` policy is not placed on the hub cluster and the policy templates do not produce any reports.
```

> If a hub cluster is managed by a global hub cluster, or if it is installed on a managed cluster instance, the self managing option might get disabled by setting the  `disableHubSelfManagement=true`. In this instance, you can still enable the `backup-restore-enabled` policy on the hub cluster. Set the `is-hub=true` label on the `ManagedCluster` resource that represents the local cluster. 

Reviewer 2: ```suggestion
If a hub cluster is managed by a global hub cluster, or if it is installed on a managed cluster instance, the self-management option might get disabled by setting the  `disableHubSelfManagement` to `true`. In this instance, you can still enable the `backup-restore-enabled` policy on the hub cluster. Set the `is-hub=true` label on the `ManagedCluster` resource that represents the local cluster. 
```

*Note:* A hub cluster is automatically imported and managed by itself by using the `local-cluster` managed cluster. If you disable this by setting `disableHubSelfManagement=true` on the `MultiClusterHub` resource, the `backup-restore-enabled` policy is not placed on the hub cluster, and the policy templates do not produce any reports.

> If a cluster hub is managed by a global hub cluster, or if it is installed on a managed cluster instance, the self managing option might be disabled by setting the  `disableHubSelfManagement=true`. In this instance, you can still enable the `backup-restore-enabled` policy on the cluster hub. Do this by setting the `is-hub=true` label on the `ManagedCluster` resource that represents the managed hub cluster. 

Reviewer: ...policy on the cluster hub. Why do we say _cluster hub_ @jc-berger ?

_might be disabled by setting the  `disableHubSelfManagement=true`_ (meaning _might already be_, or the user _can_ disable?)

Extra words here so please make concise:
_Do this by setting the_ ....  -- Just make this imperative with active voice. `Do this` is also self-referential and can be eliminated. _Set the is-hub=true label on the ManagedCluster_...( should work?)

Also:

_that represents the managed hub cluster_ -- can we just say _local cluster_ here?

While we are here, I think we say this differently in the doc, line 6:

_A hub cluster is automatically imported and managed by itself by using the `local-cluster` managed cluster._

A hub cluster that is imported and managing itself is a `local-cluster`, a hub cluster that is self managed. 

_"by using the `local-cluster` managed cluster."_ is a bit confusing to me. (??) We may want to clarify there what a local cluster is?
